coalib/output/ConfWriter.py

```diff
@@ -105,7 +105,7 @@ def __write_key_val(self, keys, val):
 
         self.__file.write((self.__key_delimiter + " ").join(keys) + " " +
                           self.__key_value_delimiter + " " + val + "\n")
-        self.__wrote_newline = False
+        self.__wrote_newline = True
```

Comment:
maybe we should just not write that newline at all? Wouldn't that be simpler? This adds the newline only in one very specific case which isn't helping a lot anymore